mobly/snippet/client_base.py

```diff
@@ -107,14 +97,13 @@ def __init__(self, package, device):
   def __del__(self):
     self.close_connection()
 
-  def start_server(self):
+  def initialize(self):
     """Starts the server on the remote device and connects to it.
```

Comment:
the docstring should be adjusted accordingly

Reply:
Done